feat: Prepare for 0.13.2 release

akash-jain-10 opened the pull request:
### Describe your changes :
<!-- Explain what you have done & tag your assigned issue !-->
- [x] Bumped up Chart Versions
- [x] Updated the repository image names to docker.getcollate.io/openmetadata/*

#
### Type of change :
<!-- You should choose 1 option and delete options that aren't relevant -->
- [x] Improvement
- [x] New feature

#
### Checklist:
<!-- add an x in [] if done, don't mark items that you didn't do !-->
- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
<!-- Please see the contributing guidelines and then add your reviewer(s) !-->
<!--- OpenMetadata community thanks you for explaining your changes in detail !-->
<!--- If you are unsure of people to review your work, you can add anyone of these developers :) !-->
<!--- @shahsank3t -->
<!--- @sureshms @harshach -->
<!--- @ayush-shah @akash-jain-10 -->

nahuelverdugo reviewed (commented):
What about the new values added to `openmetadata.yaml` file? Do we have to update them somewhere?

akash-jain-10 commented:
> What about the new values added to `openmetadata.yaml` file? Do we have to update them somewhere?

Hello @nahuelverdugo , can you help me with the new values here? I'll provision them along with this PR. Thanks!

nahuelverdugo commented:
Hi Akash after running this command:
```
git diff 0.13.1 0.13.2 -- conf/openmetadata.yaml 
```
We have changed the following lines:
Added
```
searchIndexMappingLanguage: ${ELASTICSEARCH_INDEX_MAPPING_LANG:-EN}
```
Uncommented:
```
eventMonitoringConfiguration:
  eventMonitor: ${EVENT_MONITOR:-prometheus}  # Possible values are "prometheus", "cloudwatch"
  batchSize: ${EVENT_MONITOR_BATCH_SIZE:-10}
```
So I think we need values for `EVENT_MONITOR`, `EVENT_MONITOR_BATCH_SIZE`, and `ELASTICSEARCH_INDEX_MAPPING_LANG`.

akash-jain-10 commented:
> Hi Akash after running this command:
> 
> ```
> git diff 0.13.1 0.13.2 -- conf/openmetadata.yaml 
> ```
> 
> We have changed the following lines: Added
> 
> ```
> searchIndexMappingLanguage: ${ELASTICSEARCH_INDEX_MAPPING_LANG:-EN}
> ```
> 
> Uncommented:
> 
> ```
> eventMonitoringConfiguration:
>   eventMonitor: ${EVENT_MONITOR:-prometheus}  # Possible values are "prometheus", "cloudwatch"
>   batchSize: ${EVENT_MONITOR_BATCH_SIZE:-10}
> ```
> 
> So I think we need values for `EVENT_MONITOR`, `EVENT_MONITOR_BATCH_SIZE`, and `ELASTICSEARCH_INDEX_MAPPING_LANG`.

Thanks @nahuelverdugo . I've provisioned a way in k8s helm values for the above said environment variables!
